### PR TITLE
refactor(dbt): migrate `project_fully_featured` to use `@dbt_assets`

### PR DESCRIPTION
**Pull request by rexledesma**
## Summary & Motivation
- remove unnecessary nesting of `profiles.yml` under `config/`
- use `DagsterDbtTranslator` and `@dbt_assets` instead of `load_assets_from_dbt_manifest` 

## How I Tested These Changes
- `dagster dev` locally
- existing pytest

### CODE REVIEW

**Comment by rexledesma** [comment]
Current dependencies on/for this PR:
* master
  * **PR #16394** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16394" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16394?utm_source=stack-comment).

File: examples/project_fully_featured/project_fully_featured/__init__.py [hunk 1]
```diff
@@ -2,12 +2,26 @@
...
+    recommender_assets_sensor,
+)
 from .resources import RESOURCES_LOCAL, RESOURCES_PROD, RESOURCES_STAGING
 from .sensors import make_slack_on_failure_sensor
 
-all_assets = [*core_assets, *recommender_assets, *dbt_assets, *activity_analytics_assets]
+all_assets = [
+    *core_assets,
+    *recommender_assets,
+    hacker_news_dbt_assets,
```
Comment (OwenKephart) [inline examples/project_fully_featured/project_fully_featured/__init__.py:22]: this is kinda ugly but I don't see any particularly better option

**Review by OwenKephart — APPROVED** [review]
nice

**Comment by rexledesma** [comment]
### Merge Activity

* **Sep 11, 8:57 AM**: @rexledesma started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16394).

**Pull request merged**